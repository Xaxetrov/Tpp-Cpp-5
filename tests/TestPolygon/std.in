PC Poly1 10 10 10 20 15 25 30 20 30 10
PC Poly2 -10 -10 0 0
PC Poly1 10 10 10 20 15 25 30 20 30 11
PC Poly3 0 -10 0 0 10 0 1 -1
PC Poly4 0 0 0 10 10 10 10 0 0 0 0 10
EXIT
