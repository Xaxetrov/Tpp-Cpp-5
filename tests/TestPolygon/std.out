C: PC Poly1 10 10 10 20 15 25 30 20 30 10
R: OK
C: EXIT
